packages/pn-landing-webapp/api/data/en.tsx

```diff
@@ -48,10 +48,10 @@ const onReadClick = () => {
 };
 
 const navigation: INavigationBarProps = {
-  title: "Piattaforma Notifiche",
+  title: "Notifications Platform",
```

Comment:
This title is translated into English and Slovenian, but not in French neither in German, is that on purpose?

Reply:
Good question and I can't answer. Bring this to the SU.

Reply:
We discussed this issue in the SU, the decision was to merge this PR as is and expose the details about translations in future meetings with other staff linked to PN.